hazelcast/src/main/java/com/hazelcast/jet/impl/deployment/ChildFirstClassLoader.java

```diff
@@ -33,6 +33,8 @@
  */
 public class ChildFirstClassLoader extends URLClassLoader {
 
+    private boolean closed;
```

Comment:
is concurrency an issue? should we have `volatile` ?

Reply:
The current use doesn't require it. But making it volatile doesn't hurt.